### PR TITLE
Set notification for Whatsapp by CallmeBot

### PR DESCRIPTION
**Pull request by sm622**
First, get your [CallmeBot Webhook URL](https://www.callmebot.com/blog/free-api-whatsapp-messages/), then set it in your systems environment:

export CallMeBot_Phone_No="..."     # your phone no
export CallMeBot_apikey="..." 

Ok, let's set notification hook:

acme.sh --set-notify  --notify-hook callmebotWhatsApp
The CallMeBot_Phone_No and CallMeBot_apikeywill be saved in ~/.acme.sh/account.conf and will be reused when needed.

### CODE REVIEW

**Comment by sm622** [comment]
Hello @Neilpang 
can you please help me with this? 
This is my first pull request and I want to contribute to this project.

**Comment by Neilpang** [comment]
please update and use the full upper case variable names, like:  MAILGUN_API_KEY

**Comment by Neilpang** [comment]
please also add your usage here:  https://github.com/acmesh-official/acme.sh/wiki/notify

**Review by sm622 — COMMENTED** [review]
changes done

**Comment by sm622** [comment]
Hi @Neilpang 
Thanks for your reply. Can you please check and review this pull request?

File: notify/callmebotWhatsApp.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  CALLMEBOT_API_KEY="${CALLMEBOT_API_KEY:-$(_readaccountconf_mutable CALLMEBOT_API_KEY)}"
+  if [ -n "$CALLMEBOT_API_KEY" ]; then
+    _saveaccountconf_mutable CALLMEBOT_API_KEY "$CALLMEBOT_API_KEY"
+  fi
+
+  _waUrl="https://api.callmebot.com/whatsapp.php"
+
+  _Phone_No="$(printf "%s" "$CALLMEBOT_YOUR_PHONE_NO" | _url_encode)"
+  _apikey="$(printf "%s" "$CALLMEBOT_API_KEY" | _url_encode)"
+  _message="$(printf "$CQHTTP_CUSTOM_MSGHEAD *%s*\\n%s" "$_subject" "$_content" | _url_encode)"
```
Comment (Neilpang) [inline notify/callmebotWhatsApp.sh:31]: what is CQHTTP_CUSTOM_MSGHEAD ?

Comment (sm622) [reply notify/callmebotWhatsApp.sh]: @Neilpang 
CQHTTP_CUSTOM_MSGHEAD removed.
I think no other change is needed. Please confirm.

**Pull request merged**